src/app/public/modules/fluid-grid/fluid-grid.component.ts

```diff
@@ -30,7 +30,9 @@ export class SkyFluidGridComponent {
   }
 
   /**
-   * Defines the size of the padding between each column.
+   * Specifies a `SkyFluidGridGutterSize` enum to define the size of the padding
+   * between columns. The valid options are `Small` for a 10px gutter, `Medium` for
+   * a 20px gutter, and `Large` for a 30px gutter.
```

Comment:
We'll probably want to put jsdoc comments on the `SkyFluidGridGutterSize` enum as well. In fact, if those get written, then you might not need this line here anymore `The valid options are...` ?

Reply:
Good eye on that. Made that update! 👍